# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../oprofile/gcc43.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- oprofile-0.9.3/pp/opannotate_options.cpp.vanilla	2008-04-09 11:03:34.000000000 +0000
+++ oprofile-0.9.3/pp/opannotate_options.cpp	2008-04-09 11:03:41.000000000 +0000
@@ -9,6 +9,8 @@
  * @author Philippe Elie
  */
 
+#include <stdlib.h>
+
 #include <vector>
 #include <list>
 #include <iterator>
--- oprofile-0.9.3/pp/common_option.cpp.vanilla	2008-04-09 11:03:07.000000000 +0000
+++ oprofile-0.9.3/pp/common_option.cpp	2008-04-09 11:03:13.000000000 +0000
@@ -9,6 +9,8 @@
  * @author Philippe Elie
  */
 
+#include <stdlib.h>
+
 #include <iostream>
 #include <sstream>
 #include <iterator>
--- oprofile-0.9.3/pp/oparchive.cpp.vanilla	2008-04-09 11:04:34.000000000 +0000
+++ oprofile-0.9.3/pp/oparchive.cpp	2008-04-09 11:04:40.000000000 +0000
@@ -10,6 +10,8 @@
  * @author Philippe Elie
  */
 
+#include <stdlib.h>
+
 #include <iostream>
 #include <fstream>
 
--- oprofile-0.9.3/pp/opgprof_options.cpp.vanilla	2008-04-09 11:04:19.000000000 +0000
+++ oprofile-0.9.3/pp/opgprof_options.cpp	2008-04-09 11:04:27.000000000 +0000
@@ -9,6 +9,8 @@
  * @author Philippe Elie
  */
 
+#include <stdlib.h>
+
 #include <vector>
 #include <list>
 #include <iterator>
--- oprofile-0.9.3/libpp/profile.cpp.vanilla	2008-04-09 11:01:58.000000000 +0000
+++ oprofile-0.9.3/libpp/profile.cpp	2008-04-09 11:02:07.000000000 +0000
@@ -11,6 +11,7 @@
  */
 
 #include <unistd.h>
+#include <string.h>
 
 #include <iostream>
 #include <string>
--- oprofile-0.9.3/libpp/op_header.cpp.vanilla	2008-04-09 11:01:32.000000000 +0000
+++ oprofile-0.9.3/libpp/op_header.cpp	2008-04-09 11:01:45.000000000 +0000
@@ -9,6 +9,7 @@
  * @author Philippe Elie
  */
 
+#include <string.h>
 #include <iostream>
 #include <cstdlib>
 #include <iomanip>
--- oprofile-0.9.3/libpp/sample_container.cpp.vanilla	2008-04-09 11:02:26.000000000 +0000
+++ oprofile-0.9.3/libpp/sample_container.cpp	2008-04-09 11:02:36.000000000 +0000
@@ -9,6 +9,8 @@
  * @author John Levon
  */
 
+#include <limits.h>
+
 #include <set>
 #include <numeric>
 #include <algorithm>
--- oprofile-0.9.3/libutil++/op_spu_bfd.cpp.vanilla	2008-04-09 10:53:18.000000000 +0000
+++ oprofile-0.9.3/libutil++/op_spu_bfd.cpp	2008-04-09 10:53:39.000000000 +0000
@@ -12,6 +12,8 @@
 
 #include <fcntl.h>
 #include <sys/stat.h>
+#include <stdlib.h>
+#include <string.h>
 
 #include <iostream>
 
--- oprofile-0.9.3/libutil++/bfd_support.cpp.vanilla	2008-04-09 10:49:50.000000000 +0000
+++ oprofile-0.9.3/libutil++/bfd_support.cpp	2008-04-09 10:50:10.000000000 +0000
@@ -15,6 +15,9 @@
 #include "string_manip.h"
 #include "cverb.h"
 
+#include <stdlib.h>
+#include <string.h>
+
 #include <iostream>
 #include <fstream>
 #include <sstream>
--- oprofile-0.9.3/libutil++/child_reader.cpp.vanilla	2008-04-09 10:51:30.000000000 +0000
+++ oprofile-0.9.3/libutil++/child_reader.cpp	2008-04-09 10:54:35.000000000 +0000
@@ -10,7 +10,11 @@
  */
 
 #include <unistd.h>
+#include <stdlib.h>
+#include <string.h>
+
 #include <sys/wait.h>
+#include <linux/limits.h>
 
 #include <cerrno>
 #include <sstream>
--- oprofile-0.9.3/libutil++/file_manip.cpp.vanilla	2008-04-09 10:50:39.000000000 +0000
+++ oprofile-0.9.3/libutil++/file_manip.cpp	2008-04-09 10:50:49.000000000 +0000
@@ -15,6 +15,8 @@
 #include <dirent.h>
 #include <fnmatch.h>
 #include <utime.h>
+#include <limits.h>
+#include <stdlib.h>
 
 #include <cstdio>
 #include <cerrno>
--- oprofile-0.9.3/libutil++/bfd_spu_support.cpp.vanilla	2008-04-09 10:52:25.000000000 +0000
+++ oprofile-0.9.3/libutil++/bfd_spu_support.cpp	2008-04-09 10:52:32.000000000 +0000
@@ -15,6 +15,8 @@
 #include "cverb.h"
 
 #include <stdlib.h>
+#include <string.h>
+
 #include <iostream>
 #include <fstream>
 #include <sstream>
--- oprofile-0.9.3/libutil++/cverb.cpp.vanilla	2008-04-09 10:51:01.000000000 +0000
+++ oprofile-0.9.3/libutil++/cverb.cpp	2008-04-09 10:51:09.000000000 +0000
@@ -9,6 +9,8 @@
  * @author John Levon
  */
 
+#include <string.h>
+
 #include <fstream>
 #include <iostream>
 #include <map>
--- oprofile-0.9.3/libabi/opimport.cpp.vanilla	2008-04-09 10:54:50.000000000 +0000
+++ oprofile-0.9.3/libabi/opimport.cpp	2008-04-09 10:55:05.000000000 +0000
@@ -23,6 +23,8 @@
 #include <fcntl.h>
 #include <unistd.h>
 #include <sys/mman.h>
+#include <stdlib.h>
+#include <string.h>
 
 using namespace std;
 
--- oprofile-0.9.3/libregex/demangle_symbol.cpp.vanilla	2008-04-09 10:59:56.000000000 +0000
+++ oprofile-0.9.3/libregex/demangle_symbol.cpp	2008-04-09 11:00:08.000000000 +0000
@@ -8,6 +8,8 @@
  * @author John Levon
  */
 
+#include <stdlib.h>
+
 #include "config.h"
 
 #include "demangle_symbol.h"
